### DISCUSSION TITLE
[Feature] Join Flights together for multi-leg trips

### DISCUSSION THREAD

**Discussion by palitu** [johanohly/AirTrail#28] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

(Last request for a while, i promise!  thanks for the software!) Where you need to transfer at an airport, to get to the final destination, it would be great to be able to enter all the flights in together, either on the same screen, or one after the other.
ie, Perth to Christchurch via Sydney.

This can then have trip names as well as potentially highlighted differently on the map.  I think it would also make it easier to enter in the flights.

**Comment by johanohly** [comment]
I have thought about adding something like "trips", basically a collection of flights, which could be used in a variety of ways - the most obvious being connecting flights.

**Reply by palitu** [reply]
yeah, that was how i was thinking of it too.

**Comment by Kernald** [comment]
This might have an impact on how the "visited countries" list is computed - potentially, if I only ever had stopovers in Singapore for example and never a trip actually ending there, should it count as a visited country?
